docs/docusaurus/docs/cloud/quickstarts/snowflake_quickstart.md

```diff
@@ -86,23 +88,31 @@ Create a Data Asset to define the data you want GX Cloud to access within Snowfl
 
     - **Role**: Enter your Snowflake role.
 
-    - **Authenticator**: Enter the Snowflake database authenticator that you want to use to verify your Snowflake connection. 
+6. Optional. Make the following changes:
 
-4. Optional. Clear **Create temp table** if you don't want to create a temporary database table.
+    - Clear **Create temp table** if you don't want to create a temporary database table. Temporary database tables store data temporarily and can improve performance by making queries run faster.
 
-5. Optional. Clear **Test connection** if you don't want to test the Data Asset connection.
+    - Select **Test connection** if you don't want to test the Data Asset connection. Testing the connection to the Data Asset is a preventative measure that makes sure the connection configuration is correct. This verification can help you avoid errors and can reduce troubleshooting downtime.
 
-6. Select **Table Asset** or **Query Asset** and complete the following fields:
+7. Click **Continue**.
 
-    - **Asset name**: Enter a name for the Data Asset.
+8. Select **Table Asset** or **Query Asset** and complete the following fields:
 
     - **Table name**: When **Table Asset** is selected, enter a name for the table you're creating in the Data Asset.
+    
+    - **Data Asset name**: Enter a name for the Data Asset. Data Asset names must be unique. If you use the same name for multiple Data Assets, each Data Asset must be associated with a unique Data Source.
 
     - **Query**: When **Query Asset** is selected, enter the query that you want to run on the table. 
 
-7. Optional. Select **Add table/query** to add additional tables or queries and repeat step 6.
+9. Select the **Complete Asset** tab to create Expectations and run validations on the entire Data Asset, or select the **Batches** tab to create Expectations and run validations on subsets of Data Asset records and then complete the following fields:
```

Comment:
If you reworded this in the manage_data_assets guide, you should reword it here as well.

Reply:
@Rachel-Reverie I added the edit as recommended. Thanks!